src/kontrol/kdist/cheatcodes.md

```diff
@@ -1316,6 +1378,58 @@ If the production is matched when no prank is active, it will be ignored.
         </whitelist>
 ```
 
+- `#etchAccountIfEmpty Account` - sets an Account code to a single byte '0u8' if the account is empty to circumvent the `extcodesize` check that Solidity might perform [source](https://github.com/foundry-rs/foundry/blob/b78289a0bc9df6e35624c632396e16f27d4ccb3f/crates/cheatcodes/src/evm/mock.rs#L54).
+
+```k
+    syntax KItem ::= "#etchAccountIfEmpty" Account [klabel(foundry_etchAccountIfEmpty)]
+ // -----------------------------------------------------------------------------------
+    rule <k> #etchAccountIfEmpty ACCT => . ... </k>
+         <accounts>
+           <account>
+             <acctID> ACCT </acctID>
+             <code> CODE => #bufStrict(1,0) </code>
+             ...
+           </account>
+           ...
+         </accounts>
+      requires lengthBytes(CODE) ==Int 0
+    rule <k> #etchAccountIfEmpty _ => . ... </k> [owise]
+```
+
+- `#setMockCall MOCKADDRESS MOCKCALLDATA MOCKRETURN` will update the `<mockcalls>` mapping for the given account.
+
+```k
+    syntax KItem ::= "#setMockCall" Account Bytes Bytes [klabel(foundry_setMockCall)]
+ // ---------------------------------------------------------------------------------
+    rule <k> #setMockCall MOCKADDRESS MOCKCALLDATA MOCKRETURN => . ... </k>
+         <mockCall>
+            <mockAddress> MOCKADDRESS </mockAddress>
+            <mockValues>  MOCKVALUES => MOCKVALUES [ MOCKCALLDATA <- MOCKRETURN ] </mockValues>
+         </mockCall>
+
+   rule <k> #setMockCall MOCKADDRESS MOCKCALLDATA MOCKRETURN => . ... </k>
+         <mockCalls>
+           ( .Bag
+            => <mockCall>
+                  <mockAddress> MOCKADDRESS </mockAddress>
+                  <mockValues> .Map [ MOCKCALLDATA <- MOCKRETURN ] </mockValues>
+               </mockCall>
+           )
+           ...
+         </mockCalls>
+```
+
+- `#execMockCall` will update the output of the function call with `RETURNDATA` using `#setLocalMem` and in case the function
```

Comment:
I'd split this into two sentences, e.g., something like
```suggestion
- `#execMockCall` will update the output of the function call with `RETURNDATA` using `#setLocalMem`.
In case the function did not end with `EVMC_SUCCESS` it will update the status code to `EVMC_SUCCESS`. 
```
Or you can just add a comma between two clauses.

Reply:
Thanks for all the suggestions! I just addressed them.